soc/espressif/soc.yml

```diff
@@ -0,0 +1,21 @@
+family:
+- name: espressif_esp32
+  series:
+  - name: esp32
+    socs:
+    - name: esp32
+      cpuclusters:
+      - name: procpu
+      - name: appcpu
+  - name: esp32s2
+    socs:
+    - name: esp32s2
+  - name: esp32s3
+    socs:
+    - name: esp32s3
+      cpuclusters:
+      - name: procpu
+      - name: appcpu
```

Comment:
cpu first i.e. `cpupro` and `cpuapp`

Reply:
Does Zephyr build system require cpu prefix position somehow?

Reply:
I ask this because current naming is based on ESP documentation, i.e., cpu as suffix.

Reply:
Ok if I create a top commit changing this in all files (instead of manually editing commit by commit)?

Reply:
yes that's fine